src/libs/LoginUtils.ts

```diff
@@ -75,4 +76,19 @@ function areEmailsFromSamePrivateDomain(email1: string, email2: string): boolean
     return Str.extractEmailDomain(email1).toLowerCase() === Str.extractEmailDomain(email2).toLowerCase();
 }
 
-export {getPhoneNumberWithoutSpecialChars, appendCountryCode, isEmailPublicDomain, validateNumber, getPhoneLogin, areEmailsFromSamePrivateDomain};
+function fetchSAMLUrl(body: FormData): Promise<Response | void> {
```

Comment:
NAB, but this sounds like a method that returns a URL relating to SAML
```suggestion
function postSAMLLogin(body: FormData): Promise<Response | void> {
```